client/src/App.js

```diff
@@ -4,6 +4,13 @@ import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
 
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  NavLink
```

Comment:
Same as first comment. For your app.js, you can get rid of BrowserRouter, Link, and Navlink. Make sure you're checking the warnings in your console during development.